app/src/main/res/values/strings.xml

```diff
@@ -446,12 +446,12 @@
     <!-- Reconnect settings -->
 
     <string name="pref_header_reconnect">Reconnection</string>
-    <string name="pref_title_auto_reconnect">Automatically reconnect</string>
-    <string name="pref_description_auto_reconnect">Automatically attempt reconnection after a connection loss</string>
+    <string name="pref_title_auto_reconnect">Reconnect automatically</string>
+    <string name="pref_description_auto_reconnect">Attempt reconnection automatically following a loss of connection</string>
```

Comment:
Is there a reason for changing 'a connection loss' to 'a loss of connection'? To me both sound correct, and the former is shorter.

Reply:
In my head it sounded more generic, as opposed to something that happens just the once. I think omitting "a" would actually do that.